src/midi/midi_fluidsynth.cpp

```diff
@@ -612,7 +637,10 @@ void MidiHandlerFluidsynth::Close()
 	// Reset the members
 	synth.reset();
 	settings.reset();
-	selected_font.clear();
+
+	if (current_sf2_path) {
+		current_sf2_path = {};
+	}
```

Comment:
Minor:  Do you really need the if statement?  It always ends up cleared regardless.

Reply:
Sorry, resolved by mistake then forgot about it. Yeah, the `if` is not needed, removed.